jsettlers.logic/src/jsettlers/main/JSettlersGame.java

```diff
@@ -93,6 +97,7 @@ private JSettlersGame(IGameCreator mapCreator, long randomSeed, INetworkConnecto
 		this.networkConnector = networkConnector;
 		this.playerId = playerId;
 		this.availablePlayers = availablePlayers;
```

Comment:
I introduced the PlayerSetting[] playerSettings now.
Please take a look at the code: Is it like meant it now?